media-libs/shaderc/shaderc-2017.2.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+COMMIT_ID="7a23a01742b88329fb2260eda007172135ba25d4"
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+
+inherit cmake-multilib python-any-r1 vcs-snapshot
+
+DESCRIPTION="Collection of tools, libraries and tests for shader compilation"
+HOMEPAGE="https://github.com/google/shaderc"
+SRC_URI="https://github.com/google/shaderc/archive/${COMMIT_ID}.tar.gz -> ${P}.tar.gz"
```

Comment:
Please name the distfile after the directory inside it (i.e. PN-COMMIT).

Reply:
Please give and example of distfile you have in mind.
Also note this is a regular release, upstream just don't do tags.

Reply:
`shaderc-7a23a01742b88329fb2260eda007172135ba25d4.tar.gz`

Reply:
I don't see it as a good idea. Just because upstream is lazy with tags, doesn't mean we cannot name tarballs sanely.

Reply:
If tarball extracts to a different directory than its name, it's not sane naming.